### ISSUE TITLE
Removing layers created by "serverless-layers" plugin

### ISSUE THREAD

**Issue by fyelci** [claygregory/serverless-prune-plugin#62]
It looks like passing `includeLayers: true` parameter doesn't remove the old layers created by the "serverless-layers" plugin.

Is it possible to add support for this plug in as well.

**Comment by claygregory** [comment]
That's right. There is a discussion around such a feature at #36. However, [my perspective](https://github.com/claygregory/serverless-prune-plugin/pull/36#issuecomment-965912311) remains unchanged.
